src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -458,7 +458,7 @@ function ReimbursementAccountPage({route, policy, isLoadingPolicy}: Reimbursemen
     if (USDBankAccountStep !== null) {
         return (
             <USDVerifiedBankAccountFlow
-                USDBankAccountStep={currentStep}
+                USDBankAccountStep={USDBankAccountStep}
```

Comment:
This change caused a flashing bug ["Confirm currency and country" page re-appears after address and website pages](https://github.com/Expensify/App/issues/71024#top)